app/controllers/wiki_controller.rb

```diff
@@ -36,6 +36,18 @@ def show
       end
     end
 
+    if @node && @node.has_power_tag('abtest')
+      if current_user == nil || (current_user.role != 'admin' && current_user.role != 'moderator')
+        if Random.rand(2) == 0
+          redirect_to DrupalNode.find(@node.power_tag('redirect')).path, notice: "You have been redirected to this page randomly for testing purposes. We apologize for any inconvenience caused."
```

Comment:
Don't we all try to do as much as we can? I have my exams around, but I will try my best to be engaged. Meanwhile, I will remove the notice.